apps/asap-server/test/fixtures/users.fixtures.ts

```diff
@@ -6,97 +6,91 @@ import {
   FetchUsersQuery,
 } from '../../src/autogenerated-gql/graphql';
 
-export const getGraphqlResponseFetchUsers = (): {
-  data: FetchUsersQuery;
-} => ({
-  data: {
-    queryUsersContentsWithTotal: {
-      total: 2,
-      items: [
-        getGraphQLUser(),
-        {
-          id: 'user-id-2',
-          created: '2020-09-23T20:45:22Z',
-          lastModified: '2020-10-26T15:33:18Z',
-          version: 42,
-          flatData: {
-            biography: 'some biography',
-            institution: 'some institution',
-            jobTitle: 'some job title',
-            onboarded: true,
-            orcidLastModifiedDate: null,
-            orcidLastSyncDate: null,
-            orcidWorks: [
-              {
-                doi: 'test-doi',
-                id: '987-654-321',
-                lastModifiedDate: '2020-10-26T15:33:18Z',
-                publicationDate: {},
-                type: 'BOOK',
-                title: 'orcid work title 2',
-              },
-            ],
-            reachOut: 'some reach out',
-            responsibilities: 'some responsibilities',
-            researchInterests: 'some research interests',
-            city: 'some city',
-            country: 'some country',
-            contactEmail: 'some@contact.email',
-            degree: 'some degree',
-            orcid: 'orcid',
-            avatar: null,
-            email: 'iwillbeback@arnold.com',
-            firstName: 'Arnold',
-            lastModifiedDate: null,
-            lastName: 'Schwatzneger',
-            questions: [],
-            expertiseAndResourceTags: [],
-            expertiseAndResourceDescription: 'Amazing person',
-            social: [
-              {
-                github: 'awesome',
-                googleScholar: null,
-                linkedIn: null,
-                researcherId: null,
-                researchGate: null,
-                twitter: null,
-                website1: null,
-                website2: null,
-              },
-            ],
-            teams: [
+export const getSquidexUsersGraphqlResponse = (
+  { total = 1 } = { total: 1 },
+): FetchUsersQuery =>
+  generateGraphqlFetchUsersResponse(
+    [...Array(total)]
+      .fill(getGraphQLUser())
+      .map((_, index) => getGraphQLUser({ id: `user-id-${index + 1}` })),
+  );
+export const getSquidexUserGraphqlResponse = (): FetchUserQuery => ({
+  findUsersContent: getGraphQLUser(),
+});
+
+const generateGraphqlFetchUsersResponse = (
+  items: NonNullable<FetchUserQuery['findUsersContent']>[],
+): FetchUsersQuery => ({
+  queryUsersContentsWithTotal: {
+    total: items.length,
```

Comment:
to add up to the two above comments - I would have a list fixture compose of a single element in an array and a total of 1, like this one: https://github.com/yldio/asap-hub/blob/93ac40b96a40f4a0ea366b17b1800f1661016e31/apps/asap-server/test/fixtures/calendars.fixtures.ts#L159-L161

. Should you need more you could just change it in the test (see the comment at the top of this file).

Reply:
See above. Lets discuss 👍

Reply:
It's the same as this: https://github.com/yldio/asap-hub/blob/d4b4ecbfe2ee2e204cc527965fbcbc2b44ea322a/apps/asap-server/test/fixtures/users.fixtures.ts#L9-L16
But this allows multiple items to be generated, as required by one of the tests.